source/localizable/analytics/index.html.md.erb

```diff
@@ -16,6 +16,11 @@ To successfully report your sales insights you have to:
 1. Integrate the Analytics Tracking Script into your website
 2. Declare <%= link_to 'Ecommerce', 'ecommerce' %> actions to report your data
 
+> ##### Note
+> To validate that the script is sending data properly we can provide you a link with realtime
```

Comment:
`realtime` may be misleading here, since the developer will have to refresh the integrator page.